versioned_docs/version-0.1.0/exceptions/exception-model.md

```diff
@@ -0,0 +1,115 @@
+be useful for your project:
+
+## ExceptionLog
+
+### Fillable Fields
+
+| Field Name | Description |
+| --- | --- |
+| `message` | The error message associated with the exception |
+| `type` | The type of the exception |
+| `file` | The file where the exception occurred |
+| `status` | The status of the exception log |
+| `trace` | The stack trace of the exception |
+| `request` | The HTTP request associated with the exception |
+| `line` | The line number where the exception occurred |
+| `thrown_at` | The date and time when the exception was thrown |
+| `exception_log_group_id` | The ID of the exception log group that this exception log belongs to |
+
+### Casts
+
+| Name | Description |
+| --- | --- |
+| `status` | The status of the exception log, casted to the `ExceptionLogStatus` enum |
+| `trace` | The stack trace of the exception, casted to an array |
+| `request` | The HTTP request associated with the exception, casted to an array |
+| `thrown_at` | The date and time when the exception was thrown, casted to an immutable datetime object |
+
+### Methods
+
+| Method Name | Return Type | Description |
+| --- | --- | --- |
+| `site()` | HasOneThrough | Returns a `HasOneThrough` relationship between the `ExceptionLog` model and the `Site` model through the `ExceptionLogGroup` model. This method allows you to retrieve the site associated with the exception log. |
+| `exceptionLogGroup()` | BelongsTo | Returns a `BelongsTo` relationship between the `ExceptionLog` model and the `ExceptionLogGroup` model. This method allows you to retrieve the exception log group that this exception log belongs to. |
+
+## ExceptionLogGroup
+
+### Fillable Fields
+
+| Field Name | Description |
+| --- | --- |
+| `site_id` | The ID of the site associated with the exception log group |
+| `message` | The error message associated with the exception |
+| `type` | The type of the exception |
+| `file` | The file where the exception occurred |
+| `line` | The line number where the exception occurred |
+| `first_seen` | The date and time when the exception was first seen |
+| `last_seen` | The date and time when the exception was last seen |
+
+### Casts
+
+| Name | Description |
+| --- | --- |
+| `first_seen` | The date and time when the exception was first seen, casted to an immutable datetime object |
+| `last_seen` | The date and time when the exception was last seen, casted to an immutable datetime object |
+
+### Methods
+
+| Method Name | Return Type | Description |
+| --- | --- | --- |
+| `exceptionLogs()` | HasMany | Returns a `HasMany` relationship between the `ExceptionLogGroup` model and the `ExceptionLog` model. This method allows you to retrieve the exception logs associated with this exception log group. |
+| `site()` | BelongsTo | Returns a `BelongsTo` relationship between the `ExceptionLogGroup` model and the `Site` model. This method allows you to retrieve the site associated with this exception log group. |
+
+# Create a custom Exception Log Model
```

Comment:
This title must be the same in the model sections. 

Use:
"Using a custom x Model"